plugins/PrivacyManager/lang/de.json

```diff
@@ -175,11 +175,14 @@
         "UsersOptOut": "Benutzer Opt-Out",
         "PrivacyPolicyUrl": "URL für Datenschutzrichtlinie",
         "PrivacyPolicyUrlDescription": "Ein Link zu Ihrer Datenschutzrichtlinienseite.",
+        "ImprintUrl": "URL für Impressum",
+        "ImprintUrlDescription": "Ein Link zu Ihrem Impressum.",
         "TermsAndConditionUrl": "URL für Geschäftsbedingungen",
         "TermsAndConditionUrlDescription": "Ein Link zu Ihrer Geschäftsbedingungenseite.",
         "PrivacyPolicyUrlDescriptionSuffix": "Wenn Sie etwas angeben, wird es am Ende der Anmeldeseite und auf Seiten angezeigt, auf die der '%1$s' Benutzer Zugriff hat.",
         "ShowInEmbeddedWidgets": "In eingebetteten Widgets anzeigen",
         "ShowInEmbeddedWidgetsDescription": "Wenn angewählt, wird am Ende der eingebetteten Widgets ein Link zu Ihren Datenschutzrichtlinien und zu Ihren Geschäftsbedingungen angezeigt.",
+        "Imprint": "Impressum",
```

Comment:
These are added via an automated workflow via transifex and translators that work there. It's only necessary to update the english translation when making pull requests. Can you remove these translations?